docs/api/select.md

```diff
@@ -257,6 +266,56 @@ import Migration from '@site/static/usage/v7/select/migration/index.md';
 
 Ionic uses heuristics to detect if an app is using the modern select syntax. In some instances, it may be preferable to continue using the legacy syntax. Developers can set the `legacy` property on `ion-select` to `true` to force that instance of the input to use the legacy syntax.
 
+## Accessibility
+
+### Keyboard Interactions
+
+Ionic's keyboard interactions follow the implementation patterns of the web `<select>` instead of the native iOS select for a consistent experience across all platforms.
```

Comment:
Thoughts on stating that the patterns come from radio and checkbox instead?

I think implementation patterns of the web may be too vague.

Reply:
It should be fine.
However, I do wonder if it will be a bit repetitive since it's mentioned below. As a reader, I was able to understand how the keyboard interactions were chosen especially since you link the W3 pages.

Reply:
Ok, let's use the simpler language to start and can always adjust later 👍 

Addressed with your feedback here: https://github.com/ionic-team/ionic-docs/pull/3156/commits/dce34a64e9f33b7dd25933d65d71164f16e2d9bf